http/src/main/scala/org/broadinstitute/dsde/workbench/leonardo/monitor/leoPubsubMessageSubscriberModels.scala

```diff
@@ -167,11 +167,7 @@ object LeoPubsubMessage {
     val messageType: LeoPubsubMessageType = LeoPubsubMessageType.UpdateRuntime
   }
 
-  final case class UpdateDiskMessage(diskId: DiskId,
-                                     newSize: Option[DiskSize],
-                                     newDiskType: Option[DiskType],
-                                     newBlockSize: Option[BlockSize],
-                                     traceId: Option[TraceId])
+  final case class UpdateDiskMessage(diskId: DiskId, newSize: Option[DiskSize], traceId: Option[TraceId])
```

Comment:
if `newSize` is None, is there still point of having this message?

Reply:
I guess not.. should we make `newSize` not be an Option?

Reply:
yea...make it not optional so that if we're sending this event, it's an event that we do care.

Reply:
That's a good point, maybe we should make it non-optional.
If there are multiple potential fields to update, then we would make them optional.

Reply:
should I also make it non-optional in `UpdateDiskRequest`? https://github.com/DataBiosphere/leonardo/blob/develop/http/src/main/scala/org/broadinstitute/dsde/workbench/leonardo/api/DiskRoutes.scala#L305 (I removed `diskType` and `blockSize` from here as well)

Reply:
it needs to be optional in request because it's possible that they just want to update `diskType` or `blockSize`

Reply:
Oh so I shouldn't remove `diskType` or `blockSize` from the request? the reason i removed them is because Google doesn't support updating `diskType` or `blockSize`

Reply:
would we want to allow users to update `diskType` snd `blockSize` just in our DB?

Reply:
hmmm if that's the case..yea probably should just remove them from the request and make size non optional